src/index.html

```diff
@@ -13,5 +13,159 @@
+        </div>
+        <div class="form-field">
+          Full Date of Birth:&nbsp;
+          <input
+            type="text"
+            placeholder="mm/dd/yyyy"
+            name="Age"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          I accept the terms of agreement&nbsp;
+          <input
+            type="checkbox"
+            name="Agree?"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          Registration
+        </legend>
+        <div class="form-field">
+          E-mail:&nbsp;
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="email"
+            autocomplete="off"
+          >
+        </div>
+        <div class="form-field">
+          Password:&nbsp;
+          <input
+            type="password"
+            name="Password"
+            autocomplete="off"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        <div class="form-field-cats">
+          <label>
+            Do you love cats?
+            <input type="radio" name="loveCats?">
+            Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="loveCats?"
+            >
+            No
+          </label>
+        </div>
+        <div class="color-selector">
+          <label class="form-field-color">
+              What is your favourite color?&nbsp;
+          </label>
+          <input
+            type="color"
+            name="favouriteColour"
+            value="#ff0000"
+          >
+        </div>
+        <div>
+          <label class="form-field-time">
+            What time do you go to bed?&nbsp;
+          </label>
+          <input
+            type="time"
+            name="bedtime"
+          >
+        </div>
+        <div>
+          <label class="car-selector" for="favouriteCar">
+            What is your favourite brand of car?&nbsp;
+          </label>
+          <select
+            class="cars"
+            name="favouriteCar"
+            multiple
+          >
+            <option value="Ferrari">Ferrari</option>
+            <option value="Lamborghini">Lamborghini</option>
+            <option value="Maserati">Maserati</option>
+          </select>
+        </div>
+        <div class="form-field slide-selector" >
+          How do you rate our work?&nbsp;
+          <input
+            type="range"
+            min="0"
+            max="100"
+            value="0"
+            class="slider"
+            id="myRange"
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>
+          Additional info:&nbsp;
+        </legend>
+        <div class="comments">
+          Comments:&nbsp;
+          <textarea name="comments" id="comments"></textarea>
+        </div>
```

Comment:
why do you need attribute `id="comments"` from functional perspective?
i mean you do need it but for a certain purpose which you don't utilize with the current html structure.

```suggestion
        <div class="comments">
          Comments:&nbsp;
          <textarea name="comments" id="comments"></textarea>
        </div>
```